src/relayer/RelayerConfig.ts

```diff
@@ -61,11 +67,11 @@ export class RelayerConfig extends CommonConfig {
     this.relayerDestinationChains = JSON.parse(RELAYER_DESTINATION_CHAINS ?? "[]");
 
     // Empty means all tokens.
-    this.relayerTokens = RELAYER_TOKENS
-      ? JSON.parse(RELAYER_TOKENS).map((token) => ethers.utils.getAddress(token))
-      : [];
+    this.relayerTokens = RELAYER_TOKENS ? JSON.parse(RELAYER_TOKENS).map((token: string) => getAddress(token)) : [];
+    this.exclusiveDepositors = JSON.parse(RELAYER_EXCLUSIVE_DEPOSITORS ?? "[]").map((addr: string) => getAddress(addr));
+    this.exclusiveRecipients = JSON.parse(RELAYER_EXCLUSIVE_RECIPIENTS ?? "[]").map((addr: string) => getAddress(addr));
```

Comment:
It's possible that the ENV could read an empty string instead of undefined/null

```suggestion
    this.exclusiveDepositors = JSON.parse(RELAYER_EXCLUSIVE_DEPOSITORS || "[]").map((addr: string) => getAddress(addr));
    this.exclusiveRecipients = JSON.parse(RELAYER_EXCLUSIVE_RECIPIENTS || "[]").map((addr: string) => getAddress(addr));
```

Reply:
I'm not sure I see the point of this change - wouldn't `||` make it more likely to unintentionally fall back to an empty array? `RELAYER_EXCLUSIVE_RECIPIENTS` is defined in any way then I'd argue it should be parsed, and if it's set incorrectly then we should fail immediately, rather than silently reverting to an empty array.